detectors/sPHENIX/Fun4All_G4_sPHENIX.C

```diff
@@ -321,6 +326,11 @@ int Fun4All_G4_sPHENIX(
   Enable::TRACKING_EVAL = Enable::TRACKING_TRACK && true;
   Enable::TRACKING_QA = Enable::TRACKING_TRACK && Enable::QA && true;
 
+  // only do track matching if TRACKINGTRACK is also used
```

Comment:
This should be
Enable::TRACK_MATCHING = Enable::TRACKING_TRACK && false;
Enable::TRACK_MATCHING_TREE = Enable::TRACK_MATCHING && false;
Enable::TRACK_MATCHING_TREE_CLUSTERS = Enable::TRACK_MATCHING_TREE && false;
and remove the commented out lines 61-65. Then the user can activate this here (right after setting the tracking flags)

Reply:
That is very clever! The changes are made in this next push.